docs/bugs_and_glitches.md

```diff
@@ -80,13 +81,61 @@ Some fixes are mentioned as breaking compatibility with link battles. This can b
 - [`BattleAnimCmd_ClearObjs` only clears the first 6⅔ objects](#battleanimcmd_clearobjs-only-clears-the-first-6-objects)
 
 
+## The S.S. Aqua and the ports have you white out to your cabin or either the Vermilion or Olivine Pokémon Centers even if you saved somewhere else
```

Comment:
How is this a bug or glitch? The `blackoutmod` is intentional. I expect it's either for convenience so you can quickly get back on the ship if you faint on it, or to prevent sequence-breaking or getting stuck somehow on the first trip to Kanto.

Reply:
HGSS doesn't have this, but I agree with what you're saying here. This one may be more suited to the wiki than anything else, so it's probably a good idea to revert this one.

Reply:
Moved to the wiki.